docs/concepts/architecture/cloud-controller.md

```diff
@@ -35,7 +35,7 @@ The CCM breaks away some of the functionality of Kubernetes controller manager (
  * Route controller
  * Service controller
 
-In version 1.8, the CCM currently runs the following controllers from the preceding list:
+The CCM currently runs the following controllers from the preceding list:
```

Comment:
Don't say "currently". This is called out in our [style guide](https://kubernetes.io/docs/home/contribute/style-guide/). The idea is that this topic will still be around a year from now, and the reader won't know what we mean by "current".

Reply:
@steveperry-53 Thanks for sharing the style guide, I didn't know it existed! I'll replace "version 1.8" with "version 1.9" instead.

Reply:
@steveperry-53 fixed.